src/main/connector/system/Windows/Windows.yaml

```diff
@@ -274,3 +274,26 @@ monitors:
           system.disk.io_time: $6
           system.disk.operation_time{disk.io.direction="read"}: $7
           system.disk.operation_time{disk.io.direction="write"}: $8
+  system:
+    simple:
+      sources:
+        # SerialNumber;Caption;Version;BuildNumber
+        osInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT SerialNumber, Caption, Version, BuildNumber FROM Win32_OperatingSystem
+        # Uptime
+        uptime:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT SystemUpTime FROM Win32_PerfFormattedData_PerfOS_System
+      mapping:
+        source: ${source::osInformation}

```

Comment:
TBH, I think it would be safer to make a table join on `osInformation` and `uptime`, because mapping is supposed to you 1 resulting source. Using `system.uptime: ${source::uptime}` works just by chance 😅